app.js

```diff
@@ -14,12 +14,33 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, '/views/partials'));
 
 // Add the route handlers here:
 
+
 app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then(beersFromApi => {
+    //console.log('Beers from the database: ', beersFromApi);
+    res.render('beers', {beersFromApi});
+  })
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+  .getRandom()
+  .then(randomBeer => { 
+    console.log(randomBeer);
+    res.render('random-beer', {randomBeer}); //question: could we do randomBeer[0] to directly access the object?
```

Comment:
Yes, I would just send the single and only element in the array to the front end.
In fact, I do not find it very logical that Punk API sends "random beer" as an array.